webpack-widget.js

```diff
@@ -334,7 +331,39 @@ const getLegacyWidgetBuildConfig = (config = {}) => {
 				output: `_output/${cfg.cleanName}-build-info.yml`
 			})
 		]
-	};
+	}
+
+	// conditionally add plugins to handle guides if the directory exists in /src
+	if (fs.existsSync(`${srcPath}_guides`))
+	{
+		// attach the guideStyles css to the default entry, if used
+		build.entry['guides/guideStyles.css'] = [
+			'./node_modules/materia-widget-development-kit/templates/guideStyles.scss'
```

Comment:
This is fine, but I've grown to start relying on `require()` and `require.resolve()` to locate things that are in node modules.